README.md

```diff
@@ -1 +1,145 @@
-# angular-http-readme
\ No newline at end of file
+# $http
+
+## Overview
+
+One common requirement of apps is the ability to make HTTP requests. Luckily, Angular provides us a service named `$http` to do these with ease - let's take a look!
+
+## Objectives
+
+- Describe $http
+- Use $http methods
+- Fetch data from an API
+- Update data from an API
+
+## What is $http?
+
+$http is a core Angular service that provides a simplistic API to allow us to communicate with HTTP endpoints with ease.
```

Comment:
Before we go into `$http`, can you walk the user through creating a XHR request with vanilla javascript? This is a good opportunity to review the request-response cycle that happens with an XHR request, and will help the students understand what the $http service is actually doing for them.